### ISSUE TITLE
Table with configuration file content not displayed in UCloud

### ISSUE THREAD

**Issue by vincentvanhees** [habitus-eu/HabitusGUI#25]
When using HabitusGUI to load a configuration file no parameter table is displayed on UCloud:

![image](https://user-images.githubusercontent.com/8362333/156169197-9676f70c-8156-44fb-b56b-2437cbb976db.png)

While locally on my Ubuntu machine the table is correctly displayed:

![image](https://user-images.githubusercontent.com/8362333/156169371-298956c6-8067-4c0a-a8a3-f454df1d08d7.png)

This problem exists for both GGIR and PALMSpy, so most likely this relates to the shiny code.

#### 1. Path problems inside the R code.
GGIR and PALMSpy can run based on default settings, so at least the data and output directory paths are correct. However, it may be good to check that configuration file path is valid. To investigate:
  - [x] Update app to send config file path and the result of a check whether file exists to the UI.

#### 2. R package dependencies missing, causing that data object cannot be loaded and therefore not displayed. To investigate:
- [x] Try and see whether app can display some dummy values inside the app without data loading. 
- [ ] Insert some cat() calls inside the code to help understand which parts of the code are run.

**Comment by vincentvanhees** [comment]
Configuration file specified seems to be correct: "/work/TestDataHabitus2022/configfiles/config.csv" and file.exists gives TRUE, good. So, app seems able to find the configuration file.

**Comment by vincentvanhees** [comment]
... parameters can be loaded from params object, so loading of the configuration file content is also not the problem.

**Comment by vincentvanhees** [comment]
... params object has correct dimensions one code line before it is is given to DT::renderDataTable.
Next step, try to simplify datatable argument. to understand where it goes wrong

**Comment by vincentvanhees** [comment]
.... issue seems specific to DT::renderDT and/or DT::datatable command in the modConfigServer.R function.

**Comment by vincentvanhees** [comment]
... printed params values column 1 and 2 to the UI which shows identical info between UCloud and local machine. So, looks like there is no problem with the data itself. Also, I plotted dummy data (created inside the code). There shown locally but not on cloud. Therefore, problem must be with DT functions.

**Comment by emolinaro** [comment]
I also observed another problem, maybe related: When I modify one of the parameters in the table, the all the entries in the JSON file are put back in square brackets. As a result, the run fails.

**Comment by vincentvanhees** [comment]
When I add dummy tables to the UI it works in UCloud (see print screen below). However, when I remove the dummy tables the functionality breaks, so more investigation needed. Interesting.... will continue investigating.
![image](https://user-images.githubusercontent.com/8362333/156356419-d3a25d0f-dcf1-4169-898d-6d6457ca6f8e.png)

**Comment by vincentvanhees** [comment]
This is now fixed. I had to declare a shiny dataTableOutput object on the UI side without using it on the server side.

This changes nothing to the functionality of the GUI but is needed on UCloud. It appears to be a problem with not shiny not being able to identify the required DT function, despite that I am explicitly call it with `DT::`. 

This solution is not super elegant but probably good enough for now as it allows for testing and development of GUI to continue.
